pages/rops/update/formatters/index.jsx

```diff
@@ -67,8 +67,12 @@ function ProductTestingHint() {
   return (
     <ul className="product-testing-hint">
       <li>
-        <p><Snippet>fields.productTestingTypes.options.forced_swim_test</Snippet></p>
-      </li><li>
+        <p>
+          <Snippet>fields.productTestingTypes.options.forced_swim_test</Snippet><br />
+          <span className="govuk-hint"><Snippet>fields.productTestingTypes.hints.forced_swim_test</Snippet></span>
```

Comment:
not big deal but do we use kebab-case in other places in the file? `forced_swim_test` or camelCase? It doesn't look consistent from a quick look.

Reply:
This is an existing name. The key is used in the CSV export for analysts and the acceptance criteria asked for `forced_swim_test` to be used, see [ASL-4538](https://collaboration.homeoffice.gov.uk/jira/browse/ASL-4538).